grakn-kb/src/main/java/ai/grakn/kb/internal/concept/RelationshipReified.java

```diff
@@ -71,6 +71,7 @@
      * This is because validation requires iterating over all castings anyway.
      */
     private Cache<Set<Casting>> allCastings = new Cache(Cacheable.set(), () -> lazyCastings(Collections.EMPTY_SET).collect(Collectors.toSet()));
+    private RelationshipImpl owner;
```

Comment:
I'd really like to avoid this field if possible, but can't figure out how...
It should definitely be marked `@Nullable` either way!

Reply:
There is a way. Instead of storing a reference to this field. When the actual relationship is needed I can just do:

```
relationship = vertex().tx().getConcept(getId());
```

reckon that would be better? I was never sure.